package.json

```diff
@@ -35,8 +35,8 @@
     "dinero.js": "1.9.1",
     "formik": "2.2.9",
     "history": "4.10.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "react": "18.2.0",
```

Comment:
bumping to react 18 since I'm having some issues with the `useEffect` digestion running reliably, which this seems to fix